packages/auth/loaders/User.js

```diff
@@ -24,14 +24,23 @@ module.exports = (context) => ({
         return []
       }
 
-      return context.pgdb.public.users.find({ or }).then(users => users.map(transformUser))
+      return context.pgdb.public.users.find({ or })
+        .then(users => users.map(transformUser))
     },
     null,
     (value, rows) => rows.find(row => (
       row._raw.id === value ||
       row._raw.email.toLowerCase() === value.toLowerCase()
     ))
   ),
+  byUsername: createDataLoader(usernames =>
+    context.pgdb.public.users.find({ username: usernames })
+      .then(users => users
+        .map(u => transformUser(u))
+      ),
+  null,
+  (key, rows) => rows.find(row => row.username.toLowerCase() === key.toLowerCase())
+  ),
```

Comment:
Here's something off with indentation. Did formatter kick in?

A suggestion:

```suggestion
  byUsername: createDataLoader(
    usernames =>
      context.pgdb.public.users.find({ username: usernames })
        .then(users => users.map(u => transformUser(u))),
    null,
    (key, rows) => rows.find(row => row.username.toLowerCase() === key.toLowerCase())
  ),
```